webapp/src/components/ImageFieldEditor.tsx

```diff
@@ -11,12 +11,12 @@ export interface ImageFieldEditorProps {
 }
 
 export interface ImageFieldEditorState {
-    galleryVisible: boolean;
+    galleryState: "editor" | "gallery" | "my-assets";
```

Comment:
minor nit: `currentView` maybe? `galleryState` feels like it would be giving info about the current gallery to me, not the current state of the editor